lib/kubeclient/common.rb

```diff
@@ -253,16 +254,18 @@ def watch_entities(resource_name, options = {})
       uri = @api_endpoint.merge("#{@api_endpoint.path}/#{@api_version}/#{path}")
 
       params = {}
+      format = options[:as] == :raw ? :text : :json
```

Comment:
we've been throwing "json" as name for "native hashes/arrays, without recursiveopenstruct wrapping" but it's not a very good name.  it could well be understood to mean "string in JSON format" which is exactly what "raw" is...

Reply:
I like the `as` suggestion. I think the value should be 'ros' since that's what we provide now as default and `json` is a value we could provide in the future. BTW Format is also something we could find ourselves adding in the feature e.g someone might want a yaml response from k8s